main.go

```diff
@@ -81,9 +82,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	inspect, _, err := dockerClient.ImageInspectWithRaw(context.Background(), originalImageID.String())
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to inspect the source image (%s)\n", err.Error())
+	}
+
+	originalRepoTags, err := json.Marshal(inspect.RepoTags)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to encode image's original tags as JSON (%s)\n", err.Error())
```

Comment:
We also need to `os.Exit(1)` on error here and above.

Reply:
Ah, sorry!